personio-personnel-data-api-oa3.yaml

```diff
@@ -3983,3 +4105,722 @@ components:
+      properties:
+        attribute_id:
+          type: string
+          example: first_name
+        data_type:
+          type: string
+          enum:
+            - TEXT
+        value:
+          type: string
+          example: Robert
+        employee_id:
+          type: integer
+          example: 17
+        effective_date:
+          type: string
+          example: 2021-01-01
+
+    DurationHistoricalAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: duration_123
+        data_type:
+          type: string
+          enum:
+            - DURATION
+        duration:
+          type: integer
+          example: 10
+        employee_id:
+          type: integer
+          example: 17
+        effective_date:
+          type: string
+          example: 2021-01-01
+
+    EntityHistoricalAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: department
+        entity_id:
+          type: string
+          example: 1
+        data_type:
+          type: string
+          enum:
+            - ENTITY
+        value:
+          type: string
+          example: IT
+        employee_id:
+          type: integer
+          example: 17
+        effective_date:
+          type: string
+          example: 2021-01-01
+
+    CostCenterHistoricalAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: cost_center
+        data_type:
+          type: string
+          enum:
+            - COST_CENTER
+        employee_id:
+          type: integer
+          example: 17
+        cost_centers:
+          type: array
+          items:
+            $ref: "#/components/schemas/CostCenter"
+        effective_date:
+          type: string
+          example: 2021-01-01
+
+    SalaryHistoricalAttribute:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: salary_123
+        data_type:
+          type: string
+          enum:
+            - SALARY
+        amount:
+          type: string
+          example: 12.20
+        currency_symbol:
+          type: string
+          example: $
+        employee_id:
+          type: integer
+          example: 17
+        effective_date:
+          type: string
+          example: 2021-01-01
+
+    PublicAPIException:
+      type: array
+      items:
+        $ref: "#/components/schemas/CustomReportsErrorResponse"
+
+    CustomReportsErrorResponse:
+      type: object
+      properties:
+        status:
+          type: integer
+          example: 500
+        trace_id:
+          type: string
+          example: ABCXXX
+        timestamp:
+          type: string
+          format: date-time
+          example: "2020-01-01T00:00:00Z"
+        errors:
+          type: array
+          items:
+            $ref: "#/components/schemas/ErrorDetails"
+
+    ErrorDetails:
```

Comment:
[Chore]: Please provide example values for all defined fields of this component.